normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -1,27 +1,93 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { Link } from 'react-router'
+import { destroy, reduxForm, getValues } from 'redux-form'
+import { _ } from 'underscore'
+
+import apiFetch from '../utils/apiFetch.js';
+import { parseJsonSchema, generateFieldsFromSchema } from '../utils/formSchemaHelpers.js';
+
 import ControlActions from '../actions/ControlActions.js'
-import { reduxForm } from 'redux-form'
 import composeRecipeContainer from './RecipeContainer.jsx'
-
+import ActionForm from './ActionForm.jsx'
 
 class RecipeForm extends React.Component {
   constructor(props) {
     super(props);
     this.submitForm = this.submitForm.bind(this);
+    this.getAvailableActions = this.getAvailableActions.bind(this);
+    this.changeAction = this.changeAction.bind(this);
+
+    this.state = {
+      availableActions: [],
+      selectedAction: null,
+    };
+  }
+
+  getAvailableActions(recipeId) {
+    apiFetch(`/api/v1/action/`)
```

Comment:
Nit, maybe. Should we only use backticks for things with variables in it, and use quotes otherwise? Or is it fine to use backticks here, style wise?